metricbeat/module/coredns/stats/_meta/data_request_type_event.json

```diff
@@ -0,0 +1,30 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "coredns": {
+        "stats": {
+            "dns": {
+                "request": {
+                    "type": "A"
+                }
+            },
+            "server": "dns://:53",
+            "zone": "."
+        }
```

Comment:
Shouldn't this event contain the number of requests of type `A`? :thinking:

Reply:
Fixed, thanks